docs/metric-calculation.md

```diff
@@ -1,54 +1,29 @@
 # Metric calculation
 This section describes how to calculate metrics for the datasets generated by the [OT Platform ETL](https://github.com/opentargets/platform-etl-backend).
 
+The only parameter required for run configuration is the upcoming Open Targets release identifier in the YY.MM format. Its specification is different depending on run mode:
+* When running in **pre-ETL mode,** append "_pre", for example: `export OT_RELEASE=23.12_pre`. Release metrics will be processed and uploaded under the same run ID, in this case "23.12".
+* When running in **regular post-ETL mode,** specify release as is, for example: `export OT_RELEASE=23.12`. Because several ETL runs can be made under the same output folder, release timestamp will be autodetected and added to the run ID. For example, if the run was completed on the 26th of October, the post-ETL metrics will appear in the metrics visualisation app as "23.12_2023-10-26".
```

Comment:
I think this section needs to be updated, as the date of the ETL run is inferred from the metadata file, if I'm correct.

Reply:
So this section is actually updated already; as it says, when running you only set one variable `OT_RELEASE`, and everything else is inferred based on it.

However, I agree this section isn't easy to read as it is, so I fully rewrote it in https://github.com/opentargets/ot-release-metrics/pull/25/commits/2b11ea8a5c5a79b2e65ed7de17d3af9c96404e3c

This is how it looks now:
<img src=https://github.com/opentargets/ot-release-metrics/assets/10669118/9d5ba3d8-1171-4ae6-9944-af468aad522a width=488>